rotkehlchen/db/upgrades/v42_v43.py

```diff
@@ -44,6 +46,15 @@ def _remove_coinbasepro_credentials(write_cursor: 'DBCursor') -> None:
     )
 
 
+@enter_exit_debug_log()
+def _remove_old_csv_files(data_dir: Path) -> None:
+    """Delete old csv files for airdrops"""
+    airdrops_dir = data_dir / APPDIR_NAME / AIRDROPSDIR_NAME
+    for csv_file_path in airdrops_dir.glob('*.csv'):
+        csv_file_path.unlink()
```

Comment:
This is really sensitive code here. Please handle any unlink errors, as they would crash the app mid-upgrade and that would be bad.